plugin/pkg/admission/podnodeselector/admission.go

```diff
@@ -34,12 +34,9 @@ import (
 	corelisters "k8s.io/kubernetes/pkg/client/listers/core/internalversion"
 	kubeapiserveradmission "k8s.io/kubernetes/pkg/kubeapiserver/admission"
 	"k8s.io/kubernetes/pkg/kubeapiserver/admission/util"
+	"k8s.io/kubernetes/pkg/scheduler/algorithm"
 )
 
-// The annotation key scheduler.alpha.kubernetes.io/node-selector is for assigning
-// node selectors labels to namespaces
-var NamespaceNodeSelectors = []string{"scheduler.alpha.kubernetes.io/node-selector"}
```

Comment:
I expected the only change here to be referencing the constants, adding the stable constant, and maybe unexporting this variable. If both annotations are set on a namespace, the existing logic handles it as I would expect.